ebos/ecloutputblackoilmodule.hh

```diff
@@ -903,12 +936,59 @@ public:
                         val.second *= elemCtx.simulator().model().dofTotalVolume(globalDofIdx)
                             * getValue(intQuants.porosity());
                     }
-                    else if (key.first == "BFLOWI")
-                        val.second = this->flows_[FaceDir::ToIntersectionIndex(Dir::XPlus)][waterCompIdx][globalDofIdx];
-                    else if (key.first == "BFLOWJ")
-                        val.second = this->flows_[FaceDir::ToIntersectionIndex(Dir::YPlus)][waterCompIdx][globalDofIdx];
-                    else if (key.first == "BFLOWK")
-                        val.second = this->flows_[FaceDir::ToIntersectionIndex(Dir::ZPlus)][waterCompIdx][globalDofIdx];
+                    else if ((key.first == "BPPO") || (key.first == "BPPG") || (key.first == "BPPW")) {
```

Comment:
maybe follow style change below

Reply:
> maybe follow style change below

You mean each condition on a separate line?  I've pushed an update to that effect.

Reply:
>> > maybe follow style change below
>> 
>> You mean each condition on a separate line?
>
> Exactly

Right.  I fat-fingered this PR, so #5240 has the appropriate update to do this.